sdcm/nemesis.py

```diff
@@ -1246,11 +1246,16 @@ def disrupt_mgmt_backup(self):
         for node in self.cluster.nodes:
             update_config_file(node=node, region=region[0])
         mgr_task = mgr_cluster.create_backup_task({'location': ['s3:{}'.format(bucket_location_name[0])]})
-        mgr_task.wait_for_status([TaskStatus.DONE], timeout=43200)  # timeout is 12 hours
-        task_final_status = mgr_task.wait_and_get_final_status()
-        assert task_final_status == TaskStatus.DONE, 'Task: {} final status is: {}.'.format(
-            mgr_task.id, str(mgr_task.status))
-        self.log.info('Task: {} is done.'.format(mgr_task.id))
+
+        succeeded, status = mgr_task.wait_for_task_done_status(timeout=10800)
+        if succeeded and status == TaskStatus.DONE:
```

Comment:
looks like the 2 conditions are redundant. it must to be "DONE" when it returns "true"

Reply:
sort of, as the internal mechanism could account other (future) statuses as "done"... here is a safe side, even if it didn't succeed, the ```succeeded``` will always be ```False``` for now.